olive/cache.py

```diff
@@ -100,8 +99,6 @@ def clean_pass_run_cache(pass_type: str, cache_dir: Union[str, Path] = ".olive-c
 
     This function deletes all runs for a given pass type as well as all child models and evaluations.
     """
-    assert pass_type.lower() in PASS_REGISTRY, f"Invalid pass type {pass_type}"
```

Comment:
why is this deleted?

Reply:
As the `get_non_local_resouce`(need to import resource_path) is under `cache.py`, and we also need to implement `get_local_path`(need to rely on methods in cache) which is **used broadly in Pass related files**. 

Also as the registry need to import all the detailed passes, it brings the circular imports issues. 

Then I simplify remove this check as it involves all the passes related python module.

Would live to add it back if we have more good solutions.

Reply:
I see. the lazy import make sense for now. Thanks!